Leetcode/1464.py

```diff
@@ -0,0 +1,13 @@
+# https://leetcode.com/problems/maximum-product-of-two-elements-in-an-array/
+from typing import List
+import heapq
+
+
+class Solution:
+
+    def maxProduct(self, nums: List[int]) -> int:
+        max_heap = list(map(lambda x: x * -1, nums))
```

Comment:
Is using `lambda` and `map` necessary here instead of using just using `list comprehension`?